services/wallet/api.go

```diff
@@ -465,6 +465,11 @@ func (api *API) GetTransactionEstimatedTime(ctx context.Context, chainID uint64,
 	return api.s.router.GetFeesManager().TransactionEstimatedTime(ctx, chainID, gweiToWei(maxFeePerGas)), nil
 }
 
+func (api *API) GetTransactionEstimatedTimeV2(ctx context.Context, chainID uint64, maxFeePerGas *hexutil.Big, maxPriorityFeePerGas *hexutil.Big) (uint, error) {
```

Comment:
We already have `GetTransactionEstimatedTime` but it is deprecated and should be removed (as comment says).
That means that when we remove, we will live with the `V2`-suffixed function without reason. I'd suggest giving more unique name, e.g. `GetEstimatedTimeForFees` or something like that.

Reply:
> We already have GetTransactionEstimatedTime but it is deprecated and should be removed (as comment says).

Correct, but it's still in use by the desktop app. Does the mobile app use that endpoint?

> That means that when we remove, we will live with the V2-suffixed function without reason. I'd suggest giving more unique name, e.g. GetEstimatedTimeForFees or something like that.

There are several different ways of versioning, but for us the only 2 that can be applied are either to add suffix to the function name or to add one more parameter for sending the version. Anyhow at some point, we have that planned, we will dedicate some time to clean up the wallet API (service).